cocos/base/CCDirector.h

```diff
@@ -488,6 +489,12 @@ class CC_DLL Director : public Ref
      */
     void resetMatrixStack();
 
+    /**
+     * returns the cocos2d thread id.
+     Useful to know if certain code is already running on the cocos2d thread
+     */
+    const std::thread::id& getCocos2dThreadId() const { return _cocos2d_thread_id; }
```

Comment:
@minggo 
Yes, we can ignore the `getCocosThread` in Lua and JS.
What changes should I do in order to ignore it ? Thanks!